src/usecases/refreshToken.js

```diff
@@ -0,0 +1,24 @@
+import moment from "moment";
+
+export default ({ getTokenProvider }) => (token) => {
+  const expiresAt = new Date(token.exp * 1000);
```

Comment:
Sorry, probs a silly question: What does the `token.exp * 1000` mean?

Reply:
Date is initialised with milliseconds since Epoch, but the expiry on the token is stored in seconds since Epoch, so we need to multiply by 1000 here. 

I'm going to either move the magic number to a constant to be a bit more descriptive, or move to moment with has an initialiser that uses seconds

Reply:
Ah I see. Yush, that would be good. Thanks!

Reply:
Replaced it with moment.unix()